src/ai/backend/test/cli_integration/admin/test_keypair_resource_policy.py

```diff
@@ -9,6 +9,7 @@ def test_add_keypair_resource_policy(run: ClientRunnerFunc):
 
     # Add keypair resource policy
     add_arguments = [
+        '--output=json',
```

Comment:
good!
but we also need to add `--max_session_lifetime` argument since `keypair_resource_policies` table was updated https://github.com/lablup/backend.ai-manager/pull/583/files#diff-b76576f82198e4a26749f24859e7496670741528f1868d36b1eef7a5084808d0

Reply:
Thank you for the comment!
It seems that `max_session_lifetime` is not considered in `backend.ai-client-py` yet; the latest code is below:
https://github.com/lablup/backend.ai-client-py/blob/b6d03cc0dcfb56dc51075fb0a2994fd5ea6cace0/src/ai/backend/client/func/keypair_resource_policy.py#L37

Would it be okay to add `max_session_lifetime` to `backend.ai-client-py` and then work on `backend.ai-test`?